worker/src/main/java/gov/cms/ab2d/worker/processor/JobProcessorImpl.java

```diff
@@ -60,31 +59,9 @@
     private final OptOutRepository optOutRepository;
 
 
-    @Override
-    @Transactional(propagation = Propagation.REQUIRES_NEW, isolation = Isolation.SERIALIZABLE)
-    public Job putJobInProgress(String jobId) {
-
-        final Job job = jobRepository.findByJobUuid(jobId);
-        if (job == null) {
-            log.error("Job was not found");
-            throw new IllegalArgumentException("Job " + jobId + " was not found");
-        }
-
-        // validate status is SUBMITTED
-        if (!SUBMITTED.equals(job.getStatus())) {
-            final String errMsg = String.format("Job %s is not in %s status.", jobId, SUBMITTED);
-            log.error("Job is not in submitted status");
-            throw new IllegalArgumentException(errMsg);
-        }
-
-        job.setStatus(IN_PROGRESS);
-
-        return jobRepository.save(job);
-    }
-
     @Override
     @Transactional(propagation = Propagation.NEVER)
-    public Job processJob(final String jobUuid) {
+    public Job process(final String jobUuid) {
 
         final Job job = jobRepository.findByJobUuid(jobUuid);
         log.info("Found job : {}", jobUuid);
```

Comment:
You shouldn't need to put the jobUuid here since it's already coming from JobHandler line 46 with MDC.put

Reply:
removed jobUuid from the above log statement.